style.css

```diff
@@ -0,0 +1,395 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+/* body {
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+} */
+
+header {
+  width: 100%;
+  height: 45px;
+  background-color: #fff;
+  border-bottom: 1px solid #e2e2e2;
+  top: 0;
+  position: sticky;
+  box-shadow: rgba(0,0,0,0.08) 0 0 5px 1px;
+  z-index: 5;
+}
+
+.nav-container {
+  display: flex;  
+  justify-content: space-between;
+  z-index: 10;
+}
+ 
+.nav-social {
+  margin-left: 20px;
+  display: flex;
+  padding-top: 15px;
+  font-weight: 700;
+  font-size: 14px;
+} 
+
+i {
+  margin-right: 22px;
+  font-size: 14px;
+}
+
+.title {
+  padding-top: 10px;
+  font-size: 23px;
+}
+
+.nav-btn {
+  padding-top: 5px;
+  margin-right: 20px;
+}   
+
+.btn {
+  background-color: rgb(86, 123, 149);
+  color: #fff;
+  border: 1px solid rgb(50, 104, 145);
+  font-size: 11px;
+  line-height: 11px;
+  font-weight: 700;
+  letter-spacing: 0.05em;
+  padding: 10px;
+  border-radius: 3px;
+  cursor: pointer;
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  transition: all 0.6s ease 0s;
+  white-space: nowrap;
+  vertical-align: middle;
+}
+
+main {
+  max-width: 950px;
+  margin: 0 auto;
+}
+
+h1 {
+  font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+  font-style: italic;
+  margin: 20px auto;
+  font-size: 40px;
+  width: 600px;
+}
+
+img {
+  width: 100%;
+}
+
+figcaption {
+  color: rgb(68, 68, 68);
+  font-family: georgia,'times new roman',times,serif;
+  text-align: left;
+  width: 760px;
+  margin-top: 10px;
+  font-size: 16px;
+  line-height: 1.3rem;
+}
+
+figcaption a {
+  color: #888;
+  text-decoration: none;
+  font-size: 14px;
+}
+
+.sub-main {
+  max-width: 620px;
+  margin: 20px auto;
+  /* display: flex; */
+}
+
+.article-header {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+/* article {
+  width: 70%;
+} */
```

Comment:
You comment on this code because you no longer need it. Please remove it completely.